addons/mass_mailing/models/mailing.py

```diff
@@ -177,6 +177,9 @@ def _get_default_mail_server_id(self):
         default=_get_default_mail_server_id,
         help="Use a specific mail server in priority. Otherwise Odoo relies on the first outgoing mail server available (based on their sequencing) as it does for normal mails.")
     contact_list_ids = fields.Many2many('mailing.list', 'mail_mass_mailing_list_rel', string='Mailing Lists')
+    use_exclusion_list = fields.Boolean(
```

Comment:
Should probably include in migration script so that all mailings use exclusion list by default (not sure if the default is going to be taken from the field, anyway deserves a quick field creation with default in upgrade I think).